src/Shared/CertificateGeneration/CertificateManager.cs

```diff
@@ -485,7 +485,7 @@ internal static void ExportCertificate(X509Certificate2 certificate, string path
                         char[] pem;
                         if (password != null)
                         {
-                            keyBytes = key.ExportEncryptedPkcs8PrivateKey(password, new PbeParameters(PbeEncryptionAlgorithm.Aes256Cbc, HashAlgorithmName.SHA256, 100000));
+                            keyBytes = key.ExportEncryptedPkcs8PrivateKey((ReadOnlySpan<char>)password, new PbeParameters(PbeEncryptionAlgorithm.Aes256Cbc, HashAlgorithmName.SHA256, 100000));
```

Comment:
Please add TODOs with a link to an issue for cleaning up temporary hacks in this type.